rgw/v2/tests/s3_swift/resuables.py

```diff
@@ -65,7 +66,22 @@ def upload_object(s3_object_name, bucket, TEST_DATA_PATH, config, user_info, app
     if object_uploaded_status is None:
         log.info('object uploaded')
 
+def upload_object_with_tagging(s3_object_name, bucket, TEST_DATA_PATH, config, user_info, obj_tag, append_data=False, append_msg=None):
```

Comment:
I think the only change compared to `upload_object` function is obj_tag. it would not be correct to create a new function for this. try to rework for the existing function `upload_object`

Reply:
I had tried incorporating Tagging as ExtraArgs  in the 'upload_object' method which uses the upload_file() to upload an s3 object. The issue I had faced was that sometimes the object was getting tagged with the tagSet provided and sometimes it failed. Hence had to write another method as changing the basic module would require changes to be made at other scripts too.